### ISSUE TITLE
Poor contrast on text input in darkmode (Android?)

### ISSUE THREAD

**Issue by laurieontech** [skillrecordings/products#77]
Some people are reporting black text on a black background within the sign up form fields.

This is on chrome, in one case it's only autofill on a desktop browser. In the other it's on android.

Both are using some form of dark mode, desktop is an extension and the other is "android dark theme".

Screenshots show a webkit autofill color in the desktop use case.

**Comment by joelhooks** [comment]
https://twitter.com/shashiwhocodes/status/1436447639319584773?s=20

![image](https://user-images.githubusercontent.com/86834/132925858-a45caa30-84b9-4728-9cf6-123c89c58fd1.png)

https://twitter.com/shashiwhocodes/status/1436451935561953286?s=21

![image](https://user-images.githubusercontent.com/86834/132925869-b014edb1-f000-4643-85b1-32a901dea966.png)

**Comment by laurieontech** [comment]
https://twitter.com/joshooaj/status/1436509829393125378?s=20
